pages/about.html

```diff
@@ -4,9 +4,39 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About Me</title>
+  <link href="../styles/styles.css" rel="stylesheet">
+  <link href="../styles/about.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>
+      <h1>Gabriela Ibarra</h1>
+      <nav class="nav-container">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="portfolio.html">Portfolio</a>
+      </nav>
+    </header>
+    <main>
+      <h1> About Me</h1>
+      <div class="side-by-side">
+        <img alt="Black and white image of young woman" src="../images/gibarra.JPG"/>
+        <p class="main-text"">  My first exposure to programming was through a summer research experience in which I wrote Python scripts
```

Comment:
looks like VSCode tried to help you by auto-completing the double quotes there! 

```suggestion
        <p class="main-text">  My first exposure to programming was through a summer research experience in which I wrote Python scripts
```